packages/strapi-plugin-users-permissions/admin/src/containers/Providers/index.js

```diff
@@ -219,14 +220,40 @@ const ProvidersPage = () => {
           <form onSubmit={handleSubmit}>
             <Row>
               {formToRender.form.map(input => {
+                if (input.name === 'subdomain') {
```

Comment:
Regarding the comment I have added in the utils/forms.js  file, this line won't be needed so you can remove them.